prime-router/docs/standard-operating-procedures/update-observation-mapping-table.md

```diff
@@ -0,0 +1,217 @@
+3.) Login with an identity provider
+4.) Get apikey from profile to populate parameter -k in commands below
+    a.) Navigate to https://uts.nlm.nih.gov/uts/
+    b.) Select "My Profile" in top right
+    c.) Copy value from "API KEY"
+    
+
+### Updating all RCTC valuesets from the Value Set Authority Center (VSAC)
+Use ./prime lookuptables update-mapping CLI command to update all RCTC value sets from the VSAC. Be sure to select yes when prompted to write the table or output a new local csv.
+
+Example:
+```
+./prime lookuptables update-mapping -a -n observation-mapping -k YOUR_API_KEY -e prod
+```
+options:
+```
+-e, --env <name>          Connect to <name> environment. Choose between [local|test|staging|prod]
+-s, --silent              Do not generate diff or ask for confirmation
+-a, --activate            Activate the table upon creation
+-n, --name TEXT           The name of the table to perform the operation on
+-o, --output-file=<path>  Specify a file to save the table's data as CSV  
+-d, --OID                 Provide a comma-separated list of OIDs to update         
+```
+
+### Update one of more specific valueset(S) from the Value Set Authority Center(VSAC)
+To update only specific valuesets in the table include the -d parameter and provide a comma sperated list of OIDs. This will add new values from only the specified valuesets.
+
+Example:
+```
+./prime lookuptables update-mapping -a -n observation-mapping -k YOUR_API_KEY -d comma_separated_list_of_OIDS -e prod
+```
+options:
+```
+-e, --env <name>          Connect to <name> environment. Choose between [local|test|staging|prod]
+-s, --silent              Do not generate diff or ask for confirmation
+-a, --activate            Activate the table upon creation
+-n, --name TEXT           The name of the table to perform the operation on
+-o, --output-file=<path>  Specify a file to save the table's data as CSV  
+-d, --OID                 Provide a comma-separated list of OIDs to update   
+```
+
+## Updating the table manually
+
+In the event that an updated to the RCTC introduces a new valueset, or we need to map a code that does not exists in the RCTC such as a local code or AOE, we will need to update the table manually for the time being. This is accomplished by downloading the table locally to a CSV
+file and then add or remove rows as needed. The Condition Code System for any mappings added ad-hoc (included the AOEs) should be labled as "ReportStream" since they do not come from any standardized ValueSet.
+
+*********Important, ensure that when editing the file locally, entries containing longer codes do not get converted to a different format (If editing in excel, it will attempt to use scientific notation for longer numeric values)*********
+
+### Downloading the active table
+
+When the table needs to be updated manually you should first download the active table locally before updating using the ./prime lookuptables CLI command.
```

Comment:
How do you know what the active version of the table is? Is there a query we can run to easily find that? I wonder if we should build that into the command to avoid updating old versions of the table.

Reply:
You can look in metabase to find it. I will the steps to do so